content/themes/custom/models/page-guid.php

```diff
@@ -3,58 +3,65 @@
  Template name: GUID-ohjaus
 */
 
+/**
+ * PageGuid Class
+ */
 class PageGuid extends \DustPress\Model {
 
-    // Method to get page/tips with guid and redirect to correct place in site
+    /**
+     * Method to get page/tips with guid and redirect to correct place in site
+     */
     public function init() {
+
+        // Get necessary args
         $guid = get_query_var( 'guid' );
-        if ( !$guid || trim( $guid ) === '' ) {
-            wp_redirect('/');
+        $lang = strtolower( get_query_var( 'lang', 'fi' ) );
+        if ( empty( $guid ) ) {
```

Comment:
mitä tuo toinen parametri tässä tekee?

Reply:
Se on mikä tuohon tulee defaultisti jos tuota lang parametria ei ole asetettu, aiemmin se oli myöhemmässä kohtaa koodia laitettu näin: ```if ( !$lang || trim( $guid ) === '' ) $lang = 'FI';``` mutta tuo on vain paljon lyhyempi ja selvempi laittaa noin.